src/components/Voting.tsx

```diff
@@ -106,10 +115,12 @@ function VotesTable({
             await submitBatchVotes(api, connectedAccount, accountVotes);
 
             // Clear user votes
-            const accountVotesStore = await Store.storeFor<AccountVote>(
-              Stores.AccountVote
+            const db = await open(
```

Comment:
Any reason the store abstraction was removed. I believe the store will be reused across the code so the abstraction was useful.